examples/ts-node/src/index.ts

```diff
@@ -0,0 +1,42 @@
+import { SubspaceClient, Identity } from "subspace.js";
+import { mnemonicGenerate } from "@polkadot/util-crypto";
+import * as fs from "fs";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const NODE_WS_PROVIDER = process.env.NODE_WS_PROVIDER;
+const FARMER_WS_PROVIDER = process.env.FARMER_WS_PROVIDER;
+const mnemonic = mnemonicGenerate();
+
+fs.readFile("./sample-input.jpg", {}, async (err, data) => {
+  if (err) throw err;
+  await usage(new Uint8Array(data));
+});
```

Comment:
This is a bit bachwards. There is a promise-based fs API in node where you can just `await` in it easily instead of awaiting in callbacks

Reply:
I can wrap await calls in a top-level async:

```javascript
import { promises as fs } from "fs";

(async () => {
  const data = await fs.readFile("./sample-input.jpg");
  await usage(new Uint8Array(data));
  process.exit(1);
})();
```